iaso/admin.py

```diff
@@ -414,6 +414,8 @@ class SourceVersionAdmin(admin.ModelAdmin):
 @admin.register(Entity)
 @admin_attr_decorator
 class EntityAdmin(admin.ModelAdmin):
+    search_fields = ["account__name", "entity_type__name", "attributes__json"]
```

Comment:
It could be really useful to search on `attributes__id` and `attributes__uuid` as well.

If I'm not mistaken, `attributes__uuid` will allow us to find entities by their UUIDs